idea/api-gateway/src/common/helpers/is-network-data-available-.ts

```diff
@@ -0,0 +1,5 @@
+import { dataStorageServicesMap } from '../../rabbitmq/init-rabbitmq';
+
+export function isNetworkDataAvailable(genesis: string): boolean {
```

Comment:
Is it really necessary to store this function and testbalanceAvailable in defferent files?

Reply:
not, necessary. I will delete the flag `functions` and replace with `has` method of the `testBalanceServicesMap` and `dataStorageServicesMap`.